docs/modules/ROOT/pages/references/operators.adoc

```diff
@@ -0,0 +1,233 @@
+= Operators
+
+An OpenShift 4 cluster comes with many operators and controllers preinstalled.
+This page gives an overview of all installed operators on a new cluster and their purpose.
+
+Most of the operators are described to some extent in the https://docs.openshift.com/container-platform/4.4/operators/operator-reference.html[official documentation].
+
+[cols="2,2a,2a,1", stripes=hover]
+|===
+|Description |CRDs |Namespaces |Commodore
+
+4+|**Cluster**
+
+|The https://docs.openshift.com/container-platform/4.4/operators/operator-reference.html#cluster-version-operator_red-hat-operators[Cluster Version Operator] handles updates for a cluster.
+|* `ClusterVersion` - is the configuration for automatic updates of the cluster
+|* `openshift-cluster-version`
+|
+
+|The https://docs.openshift.com/container-platform/4.4/operators/operator-reference.html#dns-operator_red-hat-operators[DNS Operator] deploys and manages CoreDNS on a cluster.
+|* `DNS.config.openshift.io/v1` - holds cluster-wide information about DNS
+* `DNS.operator.openshift.io/v1` - manages the CoreDNS component
+|* `openshift-dns-operator`
+* `openshift-dns`
+|
+
+|The https://github.com/openshift/service-ca-operator[Service CA Operator] manages the serving cert signer, ConfigMap CA bundle injector and generic CA bundle injector.
+|* `ServiceCA` - provides information to configure an operator to manage the service cert controllers
+|* `openshift-service-ca-operator`
+* `openshift-service-ca`
+|
+
+|The https://docs.openshift.com/container-platform/4.4/operators/operator-reference.html#cloud-credential-operator_red-hat-operators[Cloud Credential Operator] manages cloud provider credentials on a cluster.
+|* `CredentialsRequest` - request cloud credentials
+|* `openshift-cloud-credential-operator`
+|
+
+
+4+|https://docs.openshift.com/container-platform/4.4/architecture/control-plane.html[**Control Plane**]
+
+|The https://docs.openshift.com/container-platform/4.4/operators/operator-reference.html#openshift-apiserver-operator_red-hat-operators[OpenShift API Server Operator] installs and maintains the OpenShift API server on a cluster.
+|* `APIServer` - holds configuration shared by all API servers in the system, among them especially kube-apiserver and openshift-apiserver
+* `OpenShiftAPIServer` - provides information to configure an operator to manage openshift-apiserver
```

Comment:
That sounds weird but this might also be not being a native speaker.

> holds information the operator needs to manage …

or
 
> provides information to an operator managing …

This applies to all the other lines using a similar/same pattern.

Reply:
I just copied it from the CRD description